src/Test.sol

```diff
@@ -580,3 +580,49 @@ library stdStorage {
         return result;
     }
 }
+
+/*//////////////////////////////////////////////////////////////////////////
+                                DELTA MATHS
+//////////////////////////////////////////////////////////////////////////*/
+
+library deltaMaths {
```

Comment:
About to have some nits on naming/style, probably should make sure @ZeroEkkusu is ok with these suggestions before implementing.

First is I'd suggest renaming this library to `mathHelpers` to be a bit more generic, as maybe some other utils will get added here down the road (e.g. I can see things like `max`, `min`, etc. getting added here).

Reply:
`StdMath` is prettier than `MathHelpers`.

```solidity
a = MathHelpers.abs(x);

a = StdMath.abs(x);
```

In either case, let's pick a good, descriptive name now so we don't have to break users' code later :)

Reply:
Oh good call I like `StdMath`, consistent with the other parts of forge-std too

Reply:
Done, can resolve if happy